Deepanshu_Balani/dotnetproject/assets/api/api_new2/api_new2/DbModels/apiproject2Context.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace api_new2.DbModels
+{
+    public partial class apiproject2Context : DbContext
+    {
+        public apiproject2Context()
+        {
+        }
+
+        public apiproject2Context(DbContextOptions<apiproject2Context> options)
+            : base(options)
+        {
+        }
+
+        public virtual DbSet<Company> Company { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
+                optionsBuilder.UseSqlServer("server=CYG363; database=apiproject2; Trusted_Connection=True;");
```

Comment:
Please Explain Why the Connection String is here and Not in  AppSettings.json

Reply:
this connection string was made when automatically,i have written in startup.cs

Reply:
connection string removed